### ISSUE TITLE
Let's discuss 

### ISSUE THREAD

**Issue by azkihidayatulloh** [TIF17-Devteam/SukApps#2]
Hi guys, I have some questions, especially for you @andikarizkisyahputra 

1. Is this repo still active?
2. If yes, how is it progressing?

**Comment by azkihidayatulloh** [comment]
and another one, why use both yarn and npm in this project? @andikarizkisyahputra

**Comment by andikarizkisyahputra** [comment]
I don't think so, because it was made for the whole group because only a few passed, I saved my project in my private repository

**Comment by andikarizkisyahputra** [comment]
There are packages that only support using yarn to install, but I don't use them now 😂 lol @azkihidayatulloh
